client.go

```diff
@@ -140,6 +140,7 @@ func (c *Client) do(method, path string, body io.Reader, headers *map[string]str
 			req.Header.Set(k, v)
 		}
 	}
+	req.Header.Set("User-Agent", "go-pagerduty")
```

Comment:
It'd be good to add a version here as well, even just hardcoding `go-pagerduty/0.0.0` for the time being. Then it's at least something we can parse for.

Hardcoding because I think we need to fix how we track versioning -- the value in `main.go` doesn't match our tagged version and isn't super accessible from `client.go`.